study.md

```diff
@@ -48,7 +48,7 @@ in Rails.
 In your own words, define what the router does in Rails.
 
 ```md
-<!-- your answer here -->
+A router receives the incoming server requests (like GET or POST) and the url and hands the request off to the appropriate controller.
```

Comment:
Yes, it hands it off to the appropriate controller action. Controllers hold actions, actions determine what work is done before sending a response.